plugin/src/main/java/spp/jetbrains/sourcemarker/status/LogStatusBar.java

```diff
@@ -126,14 +140,19 @@ public Icon getIcon() {
 
         if (!scopeVars.isEmpty()) {
```

Comment:
This logic looks like it should be in VariablePattern

Reply:
resolved